LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright © 2022 headless_chrome contributors
```

Comment:
The only line I'm not 100% if it should stay like this as this should be the `<copyright holders>`
If you have better suggestions I'm open to them.